### PR TITLE
Service Images

### PR DESCRIPTION
**Pull request by summer-cook**
co-author: alisha evans <alisha@scientist.com>
co-author: summer cook <summer@scientist.com>

# Summary
- uses the `promo_image` for getOneWare & in configureServices

# Screenshots
image
image

# Notes
- we potentially still need the promo image on the "all requests" page, but this would be an update to the api endpoint `quote_groups/mine` - we may remove the image altogether on all requests for the MVP, or go a different route
- refer to [this slack thread](https://assaydepot.slack.com/archives/C03FZDALABG/p1673380114729429) for more info on the above

### CODE REVIEW

File: pages/services/[ware].js [hunk 1]
```diff
@@ -8,7 +8,6 @@ const Service = () => {
   const { ware, isLoading, isError } = useOneWare(id)
 
   if (isError) return <h1>{`${isError.name}: ${isError.message}`}</h1>
```
Comment (alishaevn) [inline pages/services/[ware].js:10]:
```suggestion
  if (isError) return <h1>{`${isError.name}: ${isError.message}`}</h1>

```

File: pages/services/[ware].js [hunk 2]
```diff
@@ -18,9 +17,10 @@ const Service = () => {
           <ItemPage
             title={ware.name}
             description={ware.description || ware.snippet}
-            // TODO(alishaevn): update the below to an actual image once
-            // https://github.com/assaydepot/scientist_api_v2/issues/184 is completed
-            img={DEFAULT_WARE_IMAGE}
+            img={{
+              src: ware.urls.promo_image || DEFAULT_WARE_IMAGE,
+              alt: `The promotional image for ${ware.name}`,
+            }}
```
Comment (alishaevn) [inline pages/services/[ware].js:23]:
``` js
export const DEFAULT_WARE_IMAGE = {
  src: item.src,
  alt: 'Several rows of test tubes with a liquid being put into one.',
}
```

due to the way `DEFAULT_WARE_IMAGE` is defined, this code will break on instances where we don't have a valid promo image.

File: utils/api/configurations.js [hunk 3]
```diff
@@ -24,7 +24,7 @@ export const configureRequests = ({ data, path }) => {
 
   return sortedRequests?.map(request => {
     const status = configureStatus(request.status)
-
+
```
Comment (alishaevn) [inline utils/api/configurations.js:27]:
```suggestion

```

**Comment by summer-cook** [comment]
@alishaevn made the requested changes, and added a tertiary so DEFAULT_WARE_IMAGE will work there

**Review by alishaevn — APPROVED** [review]

**Pull request merged**